src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -750,6 +884,21 @@ public ParameterHelper(IReadOnlyList<string> parameterNames)
 
             public object ReturnValue => _returnValue;
 
+            // Convert the parameters and their names to a dictionary
+            public Dictionary<string, object> GetParametersAsDictionary()
+            {
+                Dictionary<string, object> parametersAsDictionary = new Dictionary<string, object>();
```

Comment:
nit:

```
return _parameterNames.Zip(InvokeParameters, (name, param) => new KeyValuePair(name, param))
  .ToDictionary();
```

I like working with enumerables instead of dealing with indexes or counters if possible, saves some off by one bugs etc